dev/charts/charts.html

```diff
@@ -0,0 +1,1359 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Vaadin web components</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body>
+    <style>
+      body {
+        background: var(--lumo-base-color, #fff);
+      }
+      .theme-switcher {
+        display: flex;
+        padding: 1rem;
+        position: fixed;
+        top: 0;
+        z-index: 1;
+      }
+      .theme-swithcher button {
+        margin-right: .5rem;
+      }
+    </style>
+
+    <div class="theme-switcher">
+      <button onclick="changeTheme('')">Default theme</button>
+      <button onclick="changeTheme('gradient')">Gradient theme</button>
+      <button onclick="changeTheme('monotone')">Monotone theme</button>
+    </div>
+
+    <script>
+      //Crappy theme switcher that doesn't clear the previous theme
+      function changeTheme(theme) {
+        var charts = document.getElementsByTagName("vaadin-chart");
+        for (i = 0; i < charts.length; i++) {
+          charts[i].setAttribute("theme", theme)
+        }
+      }
+    </script>
+
+    <vaadin-chart type="area" title="Historic and Estimated Worldwide Population Growth by Region" subtitle="Source: Wikipedia.org" categories="[1750, 1800, 1850, 1900, 1950, 1999, 2050]" stacking="normal" no-legend tooltip>
+      <vaadin-chart-series title="Asia" values="[502, 635, 809, 947, 1402, 3634, 5268]" unit="Millions">
+      </vaadin-chart-series>
+      <vaadin-chart-series title="Africa" values="[106, 107, 111, 133, 221, 767, 1766]" unit="Millions">
+      </vaadin-chart-series>
+      <vaadin-chart-series title="Europe" values="[163, 203, 276, 408, 547, 729, 628]" unit="Millions">
+      </vaadin-chart-series>
+      <vaadin-chart-series title="America" values="[18, 31, 54, 156, 339, 818, 1201]" unit="Millions">
+      </vaadin-chart-series>
+      <vaadin-chart-series title="Oceania" values="[2, 2, 2, 6, 13, 30, 46]" unit="Millions">
+      </vaadin-chart-series>
+    </vaadin-chart>
+
+    <dom-bind>
+      <template preserve-content>
+        <vaadin-chart type="arearange" title="Temperature variation by day" additional-options='{
+          "chart": {
+            "zoomType": "x"
+          },
+          "xAxis": {
+            "type": "datetime"
+          },
+          "tooltip": {
+            "crosshairs": true,
+            "shared": true,
+            "valueSuffix": "°C"
+          }
+        }'>
+          <vaadin-chart-series title="Temperatures" values="[[data]]">
+          </vaadin-chart-series>
+        </vaadin-chart>
+        <iron-ajax url="./demo-data/temperature-date-min-max.json" last-response="{{data}}" auto></iron-ajax>
```

Comment:
Consider using `fetch()` to get rid of `dom-bind` and `iron-ajax` in the dev page.

Reply:
Done in all the individual examples that fetch json files for their data.